### ISSUE TITLE
Memory problem

### ISSUE THREAD

**Issue by FaisalAlneela** [HarrisonJackson/HJImagesToVideo#6]
Hello
i have used your app and i have a memory problems the app crash when the number of images is over 100 image...
is there a solution for this problem?? i don't want to be controlled with limited number of images.. 
is there another way to make video from images??

**Comment by leebeet** [comment]
Indeed, it has the memory leak in CVPixelBufferCreate() method, the memory seems can't release the buffer you create, and when u try a few more times converting or the images are too much, it crashes. Hope fix this problem, or any suggestion for images to video solution for batch images?

**Comment by fedorchukov** [comment]
You need to replace 
`return [adaptor appendPixelBuffer:buffer withPresentationTime:presentTime];`
by 
`BOOL isAppend = [adaptor appendPixelBuffer:buffer withPresentationTime:presentTime];
    CVPixelBufferRelease(buffer);
    return isAppend;`
